instrumentation/kafka-streams/src/main/java/brave/kafka/streams/KafkaStreamsTracing.java

```diff
@@ -283,6 +283,46 @@ public KafkaStreams kafkaStreams(Topology topology, Properties streamsConfig) {
     return new TracingFilterTransformerSupplier<>(this, spanName, predicate, true);
   }
 
+  /**
+   * Create a markFilter valueTransformer.
```

Comment:
Not too sold on `markFilter` as a name. The other filter implementation also marks the span as filtered or not so that would not be the distinguishing feature. Having said that, i'm also struggling to find a fitting name :/ 

`filterValues` , `filterNotValues` to indicate that it goes with a ValueTransformer perhaps, not that great either.

Reply:
yeap, I don't think naming it `filterValues` and having to actually `filter` later is a good user experience :(

Just pick `markFilter` as seems similar to current `mark` function purpose.

Reply:
@jorgheymans what about `markIsFiltered`/`markIsNotFiltered` ?